meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -16,9 +31,27 @@ SRC_URI[wl18xx-conf.md5sum] = "e0db09a1dea22b1fbcb1a5d9aa168215"
 SRC_URI[wl18xx-conf.sha256sum] = "e68e9a37995ab782faa41971704f24fd597d5abf16c47463708e90f8f080d335"
 
 do_install_append() {
+    install -d ${D}${sysconfdir}/firmware
     cp ${WORKDIR}/TIInit_11.8.32.bts ${D}${nonarch_base_libdir}/firmware/ti-connectivity/
 }
 
 do_install_append_beaglebone-yocto() {
     cp ${WORKDIR}/wl18xx-conf.bin ${D}${nonarch_base_libdir}/firmware/ti-connectivity/
 }
+
+do_install_append_imx () {
+    # Install NXP Connectivity
+    install -d ${D}${nonarch_base_libdir}/firmware/nxp
+    install -m 0644 ${WORKDIR}/imx-firmware/nxp/wifi_mod_para.conf    ${D}${nonarch_base_libdir}/firmware/nxp
+
+    # Install NXP Connectivity 8987 firmware
+    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8987/ed_mac_ctrl_V3_8987.conf  ${D}${nonarch_base_libdir}/firmware/nxp
+    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8987/sdiouart8987_combo_v0.bin ${D}${nonarch_base_libdir}/firmware/nxp
+    install -m 0644 ${WORKDIR}/imx-firmware/nxp/FwImage_8987/txpwrlimit_cfg_8987.conf  ${D}${nonarch_base_libdir}/firmware/nxp
+}
+
+PACKAGES =+ " ${PN}-nxp89xx"
+
+FILES_${PN}-nxp89xx = " \
+       ${nonarch_base_libdir}/firmware/nxp/* \
```

Comment:
I would suggest selecting the individual files instead of just using nxp/* as we might extend this recipe later on and will have to then change the selection here.